docs/buildah-manifest-add.md

```diff
@@ -36,6 +36,22 @@ the image.  If *imageName* refers to a manifest list or image index, the
 architecture information will be retrieved from it.  Otherwise, it will be
 retrieved from the image's configuration information.
 
+**--authfile** *path*
+
+Path of the authentication file. Default is ${XDG\_RUNTIME\_DIR}/containers/auth.json, which is set using `buildah login`.
+If the authorization state is not found there, $HOME/.docker/config.json is checked, which is set using `docker login`.
```

Comment:
probably ought to add at least `buildah-login(1)` if not also `docker-login(1)` to the `See Also` section at the bottom of this man page.